diff --git a/meta/recipes-devtools/binutils/binutils-cross.inc b/meta/recipes-devtools/binutils/binutils-cross.inc
index 02ec891606..adef384737 100644
--- a/meta/recipes-devtools/binutils/binutils-cross.inc
+++ b/meta/recipes-devtools/binutils/binutils-cross.inc
@@ -11,6 +11,7 @@ INHIBIT_DEFAULT_DEPS = "1"
 INHIBIT_AUTOTOOLS_DEPS = "1"
 
 SRC_URI += "file://0002-binutils-cross-Do-not-generate-linker-script-directo.patch"
+SRC_URI += "file://gold-errors-string.patch"
 
 # Specify lib-path else we use a load of search dirs which we don't use
 # and mean the linker scripts have to be relocated.
